### PR TITLE
[Fixes #39] Fix styling on admin and editor pages

### LINKED ISSUE #39: Fix mobile support
- [x] Main pages
- [ ] Admin panel
- [ ] Post editor

### PR DESCRIPTION
**Pull request by bluepichu**
Should fix the remaining points in #39.

### CODE REVIEW

**Comment by el1t** [comment]
### Side effect of being unable to scroll

Weird issue

### Forced scrolling

scrolling

**Comment by bluepichu** [comment]
I don't really think you can blame me for a virtual keyboard that takes up more or less the entire screen -.-

Suggestions on how to fix?

**Comment by el1t** [comment]
It would help if I could scroll until the chrome toolbar up top snaps offscreen, but I don't think there's a way to detect that in the page...

**Comment by el1t** [comment]
I bet it can be fixed if the header is not fixed on mobile (no scrolling anyways, so it _shouldn't_ be an issue normally), which allows the rest of the page to scroll properly instead of text appearing above the toolbar

**Comment by bluepichu** [comment]
Also, I can't really test this issue because it already works nearly-perfectly on Chrome iOS.  (Minor tweak for disabling zoom in landscape mode, maybe, but it works like a charm in portrait.)

**Comment by zwade** [comment]
Actually I think that's an artifact of Ellis' build. I don't think that's how stock android behaves

**Comment by bluepichu** [comment]
Sorry about the history rewrite - a rebase wasn't merging properly and it was the easiest way around it.

**Comment by bluepichu** [comment]
@zwade or @el1t can you confirm behavior with stock Android Chrome?

**Comment by zwade** [comment]
nm, I was wrong. apparently chrome treats text inputs different from textViews. Either way, I don't think landscape is the way to go on mobile

**Comment by el1t** [comment]
Setting header's display to `initial` instead of `fixed` like mentioned above (and removing some margin-top from main) will address the keyboard issue on the editor screen, but will obviously break the Posts screen

**Comment by bluepichu** [comment]
@el1t feel free to fix this.  (You should just be able to push to the branch to update the PR.)  I can't really do it myself since I have no device on which to test.

Also, I think you mean `static` instead of `initial`.

File: client/hbs/editor.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	{{#if user}}
 	{{set this "admin" true}}
 	{{> head.hbs}}
-	<body>
+	<body class="static">
```
Comment (bluepichu) [inline client/hbs/editor.hbs]: I'd instead add class `.editor` to the `<html>` tag and style based off of that.

Comment (el1t) [inline client/hbs/editor.hbs]: whoops only read half the comment my b

**Comment by bluepichu** [comment]
Sorry for the ridiculous delay.  LGTM, leaving the merge up to you, @el1t.

**Pull request merged**